assets/i18n/en.toml

```diff
@@ -35,7 +35,7 @@ percentTitle = "Solar Energy"
 percentSelf = "{self} kWh solar"
 percentGrid = "{grid} kWh grid"
 priceTitle = "Energy Price"
-priceFeedIn = "{feedInPrice} feed-in"
+priceFeedIn = "{feedInPrice} subsidy"
```

Comment:
Many countries do not have solar "subsidies" (subsidy would be correct term when self-consumption is additionally  subsidized)
Feed-in tariff is more common, that's the price we get for the energy that is effectively _sold to the grid_.

Reply:
Aha. I searched for it and got something else.
Will change it back.